webapp/src/types/types.ts

```diff
@@ -111,3 +111,11 @@ export type CallsUserPreferences = {
 export const CallsUserPreferencesDefault = {
     joinSoundParticipantsThreshold: 8,
 };
+
+export type EmojiData = {
+    id: string;
+    skin?: number;
```

Comment:
Can we make the skin a string instead of number? Like how the skins are represented in https://github.com/mattermost/mattermost-webapp/blob/master/utils/emoji.json
e.g. a skin looks like "1F3FB"
If not that's fine I'll make a map from number to string when rendering the emoji.

Reply:
That’s a good suggestion. However, the third party library we’re using presents it as a number

Reply:
when an emoji has a skin code, it is usually present on the unicode (that 1f3fb)
how does the library return the skin when there are multiple? like family, or holding hands ones? those usually have a skin combination as each person is represented.

Reply:
Not really sure because I haven’t read much in the documentation.

Reply:
@cyrilzhang-mm maybe we need the server to send more data about the emoji. That’s if you’ll be using this method from the library https://github.com/missive/emoji-mart/blob/2054b155cac965bb432ca6b6cd70b51304f8ed7f/packages/emoji-mart/src/utils.js#L18

Reply:
For rendering the emoji, all that is needed is the emoji name, the skin, and the unified string. Technically only the unified string is needed, note that the unified string always contains the skin as well. So it's not a problem if the skin is a number. This includes for emoji with multiple people in them, the unified string should also contain the skin for all people in the emoji. If you can check that emoji-mart does this that would be good, but I'm almost certain it will.

If you want to get an idea for how I will be rendering the emoji check my commit here https://github.com/mattermost/mattermost-plugin-calls/commit/e1cbfcb337c13727e78c08d096a38a3e0025eb0b I'm using the `EmojiIndicesByAlias` map but there is also `EmojiIndicesByUnicode` which uses the unified string instead of the emoji name. 

Also @Willyfrog let me know your thoughts as you will be rendering the emoji in the same way I think?

Reply:
I'll happily adapt to whatever we use. My only concern on they way that it is being rendered is that we need to import all the emoji data from webview and that needs updating once it updates on webapp. But other than that the system works, so I'm ok with that.